### PR TITLE
I620 smvpytest args

### PR DESCRIPTION
**Pull request by laneb**
Addressing #620. @jacobdr please have a look.

### CODE REVIEW

File: src/main/python/smv/helpers.py [hunk 1]
```diff
@@ -341,6 +341,10 @@ def smvJoinMultipleByKey(self, keys, joinType = 'inner'):
         jdf = self._jPythonHelper.smvJoinMultipleByKey(self._jdf, smv_copy_array(self._sc, *keys), joinType)
         return SmvMultiJoin(self._sql_ctx, jdf)
 
+    def topNValsByFreq(self, n, col):
```
Comment (AliTajeldin) [inline src/main/python/smv/helpers.py]: Missing pydoc description

File: src/main/scala/org/tresamigos/smv/SmvDFHelper.scala [hunk 2]
```diff
@@ -389,7 +388,7 @@ class SmvDFHelper(df: DataFrame) {
    *
    * and the following call:
    * {{{
-   *   df.dedupByKey("id")
+   *   df.debupByKey("id")
```
Comment (AliTajeldin) [inline src/main/scala/org/tresamigos/smv/SmvDFHelper.scala]: why rename `dedupByKey` to `debupByKey`? What is a `debup`?

Comment (laneb) [inline src/main/scala/org/tresamigos/smv/SmvDFHelper.scala]: I have no idea how this got changed. @Khorkhe at some point mentioned finding some  `debupByKey` typos in the documentation, but I don't know how I could've made this change.

File: src/main/python/smv/helpers.py (hunk 1 above)
Comment (laneb) [inline src/main/python/smv/helpers.py]: Must have committed this to both branches I guess. Strange. I'll document `topNValsByFreq` on the skewjoin branch and merge it there.

**Pull request merged**